app/models/manageiq/providers/openstack/network_manager/refresh_parser.rb

```diff
@@ -204,12 +203,18 @@ def find_device_object(network_port, subnet_id)
       nil
     end
 
+    def parse_cloud_subnet_network_ports(cloud_subnet_network_port)
```

Comment:
sorry to be picky, but shouldnt this be singular? `parse_cloud_subnet_network_port`